README.md

```diff
@@ -9,6 +9,8 @@ npm i appinsights-logger --save
 ### Environment setup
 Just set ENV variable `APPINSIGHTS_INSTRUMENTATIONKEY`
 
+Set the AI_MESSAGE_NAMESPACE ENV variable to identify message source. Format is [goPuff].[AgileTeam].[Repo] (gopuff.Architecture.Appinsights)
```

Comment:
This is awesome. Can we change to?
```
Format is [orgName].[AgileTeam].[Repo] (myCompany.Architecture.Appinsights)
```
Just trying to keep the public readme as generic as possible.